src/components/public-service-details-page/index.tsx

```diff
@@ -253,14 +254,28 @@ const PublicServiceDetailsPage: FC<Props> = ({
       relationType: translations.relatedBy
     }));
 
-  const getLink = (uri: string, resources: PublicServiceLegalResource[]) => {
-    const match = resources.find(resource => resource.uri === uri);
+  const getLink = (
+    uri: string,
+    resources: {
+      uri: string;
+      dctTitle?: Partial<TextLanguage>;
+      name?: Partial<TextLanguage>;
+    }[]
+  ) => {
+    const match = resources.find(
+      (resource: { uri: string }) => resource.uri === uri
+    );
 
     if (match) {
       const index = resources.indexOf(match);
       return (
-        <ScrollLink to={`${match.dctTitle}-${index}`} smooth isDynamic spy>
-          {translate(match.dctTitle)}
+        <ScrollLink
+          to={`${translate(match.dctTitle) ?? translate(match.name)}-${index}`}
+          smooth
+          isDynamic
+          spy
+        >
+          {translate(match.dctTitle ?? match.name)}
```

Comment:
sikkert greit å også ha med uri som backup her, er plutselig en ressurs uten navn som skal linkes til

Reply:
Godt poeng!

Reply:
Fikset! Er det greit å bruke uri som id for å scrolle? Vil vell evt. få samme problem der dersom ressursen ikke har navn

Reply:
Enn så lenge vil det være mulig at det dukker opp ressurser uten uri'er, men vi kommer til å legge til skolemisering, som jeg akkurat lagde pr for i begrepshøsteren. Så om ikke så alt for lenge vil det alltid være en unik uri tilgjengelig, dvs det er greit å bruke uri som scroll-id